src/main/java/at/hannibal2/skyhanni/features/fishing/FishingTimer.kt

```diff
@@ -179,6 +179,10 @@ object FishingTimer {
             IslandType.PRIVATE_ISLAND -> config.forStranded.get() && LorenzUtils.isStrandedProfile
             else -> false
         }
+        
+        if (config.showAnywhere.get() == true) {
+            rightLocation = true
+        }
```

Comment:
adding a return and moving this to above the when block could also not be a bad idea

```suggestion
        if (config.showAnywhere == true) {
            rightLocation = true
        }
```